doc/api/http.markdown

```diff
@@ -816,6 +816,16 @@ Example:
     //   accept: '*/*' }
     console.log(request.headers);
 
+When headers are duplicated in the raw headers, the duplicates are either
```

Comment:
How about the following...

Duplicates in raw headers are handled in the following ways, depending on the header name:

- Duplicates of the headers `content-types`, `content-length`, ..., are discarded.
- `set-cookie`is *always* an array. Duplicates are added to the array.
- For all other headers, the values are joined together with `', '`.